src/pydvl/influence/inversion.py

```diff
@@ -31,15 +27,16 @@ class InversionMethod(str, Enum):
 
     Direct = "direct"
     Cg = "cg"
+    Lissa = "lissa"
 
 
 def solve_hvp(
     inversion_method: InversionMethod,
     model: TwiceDifferentiable[TensorType, ModelType],
-    x: TensorType,
-    y: TensorType,
+    training_data: DataLoaderType,
     b: TensorType,
     lam: float = 0,
+    inversion_method_kwargs: Dict[str, Any] = {},
```

Comment:
wouldn't using `**kwargs` work as just well and be a bit more readable?

```suggestion
    **kwargs: Any,
```